heron/uploaders/src/java/com/twitter/heron/uploader/docker/sample.yaml

```diff
@@ -0,0 +1,11 @@
+# uploader class for transferring the topology jar/tar files to storage
+heron.class.uploader:         com.twitter.heron.uploader.docker.DockerUploader
+
+#base image
+heron.uploader.docker.base:                      ubuntu:latest
```

Comment:
Do you mean the sandbox in aurora? AFAIK you dont have a mount point inside the docker container into the aurora sandbox and any command you setup in aurora will be run into the docker container itself which means that you need to be able to get to the binaries inside the docker container.